### PR TITLE
Update badges

### PR DESCRIPTION
**Pull request by ronen**
Update the badges for rprt.  Also added coveralls badge (which required adding coveralls to the travis build)

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -6,3 +6,8 @@ require 'rspec/core/rake_task'
 RSpec::Core::RakeTask.new(:spec) do |spec|
   spec.rspec_opts = '-Ispec'
 end
+
+require 'coveralls/rake/task'
+Coveralls::RakeTask.new
+task :spec_with_coveralls => [:spec, 'coveralls:push']
+
```
Comment (paresharma) [inline Rakefile]: Not that it matters, but an extra line here.

**Comment by paresharma** [comment]
Looks good to me. :+1:

File: Rakefile (hunk 1 above)
Comment (ronen) [inline Rakefile]: > Not that it matters, but an extra line here.

Nice catch! :)

I removed the line

paresh brahmacharimayum wrote:

**Comment by paresharma** [comment]
I'll merge this branch and also will rebase https://github.com/rprt/rspec-page-regression/tree/v1.0 and push it.

**Comment by ronen** [comment]
Thanks!

**Pull request merged**